src/util/session-id.js

```diff
@@ -0,0 +1,121 @@
+/**
+ *
+ *    Copyright (c) 2025 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+import { v4 as uuidv4 } from 'uuid'
+import { parse as querystringParse } from 'querystring'
+
+// Session handling constants
+const SESSION_KEY = 'session_uuid'
+const APP_ID_KEY = 'stsApplicationId'
```

Comment:
What exactly is an appId?
If a user opens 2 Z3Light applications then do we have 2 appIds for it or the same app Id?
Similarly what exactly is an Iframe in terms of a sample application?

Reply:
"appId" is really a unique identifier for each "iframe". When an IDE opens multiple copies of ZAP in multiple iframes, each one gets one of those "appId" unique keys.

"iframe" in that context, is basically a separate ".zap" file. If you open 3 .zap files, you will have 3 iframes.